guides/CorrectingTrackLabellingSteps.md

```diff
@@ -0,0 +1,187 @@
+- It is very lightweight, browser-based and can be run offline.
+- If you download the tool locally, launch it by selecting the `via.html` file from the expanded zip archive
+
+> [!IMPORTANT]
+>
+> We recommend using Firefox or Edge to use the VIA tool.
+
+### 2. Define the project settings
+
+If we are launching the tool for the first time for a new project:
+
+- Click on the gear icon on the top bar to go to `Settings`
+- Edit the following settings:
+
+  - `Project Name`: use the name of the parent directory (i.e., the name of the clip) and add `_corrected_`+ your initials as a suffix. For example, for Sofia and the clip `09.08.2023-01-Right-Triggered1.mov`:
+    ```
+    09.08.2023-01-Right-Triggered1_corrected_SM
+    ```
+  - `Default Path`: this will be the starting point of the file explorer when loading the images to label, so we want the folder with the frames we want to label here. For the example above, the default path to input would be:
+  <!-- ToDO -->
+
+  ```
+  /Volumes/zoo/users/sminano/crabs_tracks_label/09.08.2023-01-Right-Triggered1/
+  ```
+
+> [!IMPORTANT]
+>
+> A trailing slash is needed!
+
+- Leave the rest of the settings with their default values
+- Click `Save`
+- NOTE: We found the default path bit doesn't work very reliably, so sometimes you will need to browse the image to label and select them manually.
+
+### 3. Load the data to label
+
+- Import the csv file with the tracks for this clip: `Annotations` > `Import annotations (from csv)`
+
+  - It will look for the image files in the same directory as the csv.
+
+- If the frames aren't loaded automatically:
+
+  - On the left hand side, find the `Project` section and click on the `Add Files` button
+  - Select the frames to load. Multiple selection is allowed, and also across folders at the same level (at least in Mac 🍎)
+
+- Once loaded, use the left/right arrows to navigate through the images
+
+### 4. Correcting the tracks
+
+- Now you see all the images and the bounding boxes with a bounding box number.
+
+  - Notice that when we flick through the frames, the numbers on the bounding boxes change! This is because this is not the track ID.
+
+- Make sure the track id is shown instead of the bounding box number.
+
+  - To do so, press the up key. The Up key toggles between `region label cleared `or `region label set to region attribute [track]`. To show the track ID, we want the second one `region label set to region attribute [track]`.
+
+> [!TIP]
+> Pressing the bottom key will change the colour of the bounding box, but notice that the colour is not consistent across frames. However, it may be helpful to disentangle boxes when they are overlapping.
+
+#### 4.1 Recommended approach:
+
+> [!CAUTION]
+> Remember there is no undo action, so make sure you save often, especially before correcting ID switches.
+
+- Select a crab ID to follow across frames.
```

Comment:
Just confirming that I should repeat this process for **every single crab/track ID**, across all frames within zoo/users/sminano/crabs_tracks_label/<clip>?

Reply:
To start with just have a go at the sample clip in `04.09.2023-04-Right_RE_test`, and we see how long it takes and if we can speed up the process.